src/Symfony/Component/Notifier/Bridge/FakeSms/README.md

```diff
@@ -14,10 +14,17 @@ where:
  - `FROM` is email who send SMS during development
 
 To use a custom mailer transport:
+
 ```
 FAKE_SMS_DSN=fakesms+email://mailchimp?to=TO&from=FROM
 ```
 
+#### DSN example for logger
+
+```
+FAKE_SMS_DSN=fakesms+logger://default
+```
+
```

Comment:
to log to a dedicated channel.... 

```
 FAKE_SMS_DSN=fakesms+logger://default?channel=sms
 ```

Reply:
@OskarStark yes obviously, but how to get the proper logger of this channel?

this is my point on the PR desc : `For the part an optional channel, how can we get to here? dymanically retreiving the proper logger based on the dsn config?`

ie in this part: `->replaceArgument('$logger', new Reference('logger'));` 

thank you for the fast review :)

Reply:
Indeed, we don't have this info.

So shall we keep it simple and not add a `channel` at all?

Reply:
maybe we can leverage the log context if needed

but as the [profiler offers an helper to filter on log level](https://symfony.com/blog/new-in-symfony-4-2-profiler-log-filter), maybe default logger and log level is enough for now

Reply:
> thus, are there still works to do on this transport?

No, it looks good

Reply:
Please check my comments

Reply:
done